### PR TITLE
[CIS-1007] Fix deleted messages appearance

### PR DESCRIPTION
**Pull request by SagarSDagdu**
### What does this PR do
- This PR fixes the appearance of deleted messages.

Before             |  After
:-------------------------:|:-------------------------:
  |

### CODE REVIEW

**Review by evsaev — APPROVED** [review]
changelog and LGTM 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix issue when messages were changing their sizes when channel is opened [#1260](https://github.com/GetStream/stream-chat-swift/pull/1260)
 - Fix over fetching previous messages [#1110](https://github.com/GetStream/stream-chat-swift/pull/1110)
 - Fix an issue where multiple messages in a channel could not quote a single message [#1264](https://github.com/GetStream/stream-chat-swift/pull/1264)
+- Fix deleted messages appearance [#1267](https://github.com/GetStream/stream-chat-swift/pull/1267)
```
Comment (VojtaStavik) [inline CHANGELOG.md:22]: Please add it to the `Upcoming` section

**Review by VojtaStavik — APPROVED** [review]
Can you also add a snapshot test that will verify this behavior?

File: CHANGELOG.md (hunk 1 above)
Comment (SagarSDagdu) [reply CHANGELOG.md]: Moved it to the upcoming section

**Pull request merged**